src/Aqovia.Utilities.SagaMachine/StatePersistance/RedisKeyValueStore.cs

```diff
@@ -8,7 +10,7 @@ namespace Aqovia.Utilities.SagaMachine.StatePersistance
 {
     public class RedisKeyValueStore : IKeyValueStore, IDisposable
     {
-        private const double DefaultLockExpiryTime = 500;
+        private const double DefaultLockExpiryTime = 30000;
```

Comment:
Good points but change already merged. Maybe create (an) issue(s) to resolve? (Unless it's easier to just propose a new PR - which I suspect it might be in at least the first case.